content/workshops/cascading-styles-deep.md

```diff
@@ -34,68 +35,116 @@ events:
     date: 2023-09-06
     adr: Freiburg, Germany
     url: https://smashingconf.com/freiburg-2023/workshops/miriam-suzanne/
+# placeholder media
+intro:
+  media:
+    - iframe: https://player.vimeo.com/video/691514550
+      span: full
+  caption: |
+    Join Miriam for a deep dive
+    on developing **resilient and delightful experiences**
+    that hold up across browsers, languages, and device interfaces.
```

Comment:
Looks like this isn't used (yet)?

Reply:
Yeah, we can remove for now. We need to record a new video, and don't want to wait for it to go live.